code/style.css

```diff
@@ -1,7 +1,210 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* { box-sizing: 
+  border-box; margin: 0
+ } 
+body {
+  
+  font-family: sans-serif; 
+}
+
+.logo{
+  display: none;
+}
+.navigation{
+  display: flex;
+  justify-content: space-evenly;
+  padding-top: 20px;
+  align-items: center;
+}
+.navigation-link{
+  color: rgb(197, 102, 39);
+ 
+}
+.hero {
+  /* Sizing */
+  width: 100%;
+  height: 100vh;
+
+
+/* Flexbox stuff */
+display: flex;
+   justify-content: center;
+   align-items: center;
+
+ /* Text styles */
+ text-align: center;
+}
+
+.hero::after{
+  width: 100%;
+  height: 100%;
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: rgba(23, 24, 18, 0.3);
+}
+
+  /* Background styles */
+  video {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    position: Absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: -1;
+    background-attachment:fixed;
+    margin: 0;
+}
+
+  .hero h1 {
```

Comment:
Why not only use h1?

Reply:
True! I think i used it so it would be easier to calssify that its only styleing the hero's h1 but i dont have any other h1's in my document so it would be good either way